src/core/qgsgpkgflusher.h

```diff
@@ -39,7 +40,23 @@ class QgsGpkgFlusher : public QObject
     explicit QgsGpkgFlusher( QgsProject *project );
     ~QgsGpkgFlusher();
 
+    /**
+     * Immediately flushes all currently scheduled tasks and returns. After stopping the flusher, requesting a new flush will be ignored.
+     */
+    void stop();
+
+    /**
+     * Reenables requests for regular flushing.
+     */
+    void start();
```

Comment:
To use it from QML (which I guess is the aim since it's registered) this will either need to be `Q_INVOKABLE` or a `Q_PROPERTY`. I prefer the second one because it's easier to bind to it.

Reply:
I want to pass the instance to a QML component defined in c++ (qfcProjectModel), but since it is constructed in QML, I find this the easiest way to do it. I will not call start/stop directly from QML for now.

Reply:
Makes sense :+1: